docs/application/native/guides/internationalization/i18n.md

```diff
@@ -1992,6 +1995,9 @@ To generate and manage an alphabetic index:
      ```
      i18n_alpha_idx_add_labels(alpha_idx, "ko", "KR");
      ```
+     > **Note**  
+     > This function is deprecated from Tizen 5.0. Instead use `i18n_alpha_idx_add_labels_with_locale_id()`
+     > which takes the locale ID parameter. To set the language as ko and country as KR, use the locale ID ko_KR.
```

Comment:
Instead, use `i18n_alpha_idx_add_labels_with_locale_id()`, which takes the locale ID parameter. To set the language as "ko" and country as "KR", use the locale ID "ko_KR".